torch/distributed/fsdp/api.py

```diff
@@ -117,6 +117,9 @@ class MixedPrecision:
             gradients back to the full parameter precision after the backward
             pass. This may be set to ``False`` to save memory if using custom
             optimizers that can perform the optimizer step in ``reduce_dtype``.
+            (Default: ``False``)
+        apply_to_forward_inputs (bool): Apply ``param_dtype`` to forward
```

Comment:
nit: I am wondering if maybe we should call this `cast_forward_inputs`?

Though, I totally understand what you mean by `apply_to_forward_inputs` -- i.e. "are we applying mixed precision to the forward inputs?".

This is just a suggestion to consider. I think `cast_forward_inputs` might be more explicit.

---

```suggestion
        cast_forward_inputs (bool): Cast floating point tensors in the forward arguments and keyword arguments to ``param_dtype``.
```